x-pack/filebeat/input/awss3/input.go

```diff
@@ -62,13 +95,54 @@ func (in *s3Input) Test(ctx v2.TestContext) error {
 }
 
 func (in *s3Input) Run(ctx v2.Context, pipeline beat.Pipeline) error {
-	collector, err := in.createCollector(ctx, pipeline)
+	var err error
+	var collector s3Collector
+
+	persistentStore, err := in.store.Access()
+	if err != nil {
+		return sderr.Wrap(err, "Can not access persistent store")
```

Comment:
Is it required to use `"github.com/urso/sderr"` here? Same as Line 109.

Reply:
I didn't realise I introduced it, moved to `fmt.Errorf` like the rest